ci/ansible/roles/pulp-backup/tasks/main.yaml

```diff
@@ -0,0 +1,52 @@
+# rhel6 pulp 2.11.z backup
+- name: Stop services before backup
+  service: "name={{ item }} state=stopped"
+  with_items:
+    - httpd
+    - pulp_celerybeat
+    - pulp_resource_manager
+    - pulp_streamer
+    - pulp_workers
+    - qpidd
+  notify: Restart services
+
+- block:
+    - name: Create backups folder
+      file: path={{ remote_bkup_dir }} state=directory
+
+    - name: Create local backups folder
+      local_action: file path={{ local_bkup_dir }} state=directory mode=0777
+      when: fetch_to_localhost
+
+    - name: Dump mongodb database
+      command: "mongodump --db pulp_database --out {{ remote_bkup_dir }}/{{ db_to_bkup.mongodb.dir }}"
+
+    - name: Archive filesystem backups
+      command: "tar -cjf {{ remote_bkup_dir }}/{{ item.value.file }}.tar.bz2 {{ item.value.src_dir }}/"
```

Comment:
I ended up using tar directly here because, from what I could find, no way to pass these options to the archive module at the moment and the standard options it uses don't preserve symlinks (needed for the published content)